elements/howto-menu/demo.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+
+<style>
+  /* The tooltip is by default unstyled. */
+  howto-menu {
+    position: absolute;
+    display: block;
+    width: 10em;
+  }
+
+  [role="menuitem"] {
+    display: block;
+    width: 100%;
+    padding: .2em .5em;
+    border: 1px solid lightgrey;
+  }
+
+  /* The menu is closed. */
+  howto-menu[aria-hidden="true"] {
+    display: none;
+  }
+</style>
+
+<div class="text">
+   <button id="menu-btn">open menu</button>
+   <howto-menu aria-labelledby="menu-btn">
```

Comment:
The goal with the label is to help the user understand the purpose of the menu. If you want to continue using a button I'd suggest changing the text to `"options"` instead of `"open menu"`.